tests/unittest/test_breakpad_resource.py

```diff
@@ -446,6 +487,22 @@ def test_submit_crash_report(self, client):
             "version": 2,
         }
 
+        mm.assert_histogram("socorro.collector.breakpad_resource.crash_size", value=632)
```

Comment:
None of these tests assert anything about metrics which is bad. I added this section here to this test to assert what metrics should get emitted.